.github/workflows/napi.yml

```diff
@@ -175,6 +180,110 @@ jobs:
       - name: Test bindings
         run: pnpm test
 
+  test-linux-x64-gnu-binding:
+    name: Test bindings on Linux-x64-gnu - node@${{ matrix.node }}
+    needs:
+      - build
+    defaults:
+      run:
+        working-directory: ./napi
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - "18"
+          - "20"
```

Comment:
current LTS version is "22" in case you want to add or update to that, just an FYI